code/modules/mob/living/living.dm

```diff
@@ -1280,18 +1280,19 @@
 	..()
 	update_z(new_z)
 
-/mob/living/MouseDrop(mob/over)
+/mob/living/MouseDrop_T(atom/dropping, atom/user)
+	var/mob/living/U = user
+	if(isliving(dropping))
+		var/mob/living/M = dropping
+		if(M.can_be_held && U.pulling == M)
+			M.mob_try_pickup(U)//blame kevinz
+			return//dont open the mobs inventory if you are picking them up
 	. = ..()
-	var/mob/living/user = usr
-	if(!istype(over) || !istype(user))
-		return
-	if(!over.Adjacent(src) || (user != src) || !canUseTopic(over))
```

Comment:
Why did you remove these checks.

Reply:
they are handled in mousedrop_t and the pulling check

Reply:
I don't follow.

Reply:
all of the checks that i removed there are already handled by the checks in mousedrop and the ability to drag someone.